aspnetcore/fundamentals/host/platform-specific-configuration.md

```diff
@@ -141,6 +141,45 @@ When building an `IHostingStartup` project, the dependencies file (*\*.deps.json
 
 Only part of the file is shown. The assembly name in the example is `StartupEnhancement`.
 
+## Configuration provided by the hosting startup
```

Comment:
"There are two approaches to configuration depending on if you want the app's config or the hosting startup's config to take precedence.   1. ..."